### ISSUE TITLE
Command `librewolf` launch tor-browser instead when another `tor-browser --allow-remote` instance exist

### ISSUE THREAD

**Issue by amalgame21** [NixOS/nixpkgs#354033]
### Describe the bug

Starting from tor-browser 14.0, if an instance `tor-browser --allow-remote` is already launched, type `librewolf` command will launch tor-browser instead of librewolf.
Both `librewolf` and `tor-browser` are installed from `nixos-unstable` branch to keep up with updates.

It does not affect `librewolf-unwrapped` though, but I cannot launch librewolf in application launcher with it.

Tested tor-browser version: 14.0 and 14.0.1
Tested librewolf version: 131.0.3

### Steps To Reproduce
1. Install both `librewolf` and `tor-browser` package from `nixos-unstable` branch
2. Type command `tor-browser --allow-remote` to launch tor-browser
3. Type command `librewolf`
4. Above step launch tor-browser instead

### Expected behavior
command `librewolf` should launch librewolf no matter what condition.

### Metadata

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.53-hardened1, NixOS, 24.05 (Uakari), 24.05.20241104.d063c1d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.8`
 - channels(root): `"home-manager-23.11.tar.gz, nixos-23.11, nixos-unstable"`
 - nixpkgs: `/home/user/.nix-defexpr/channels/nixpkgs`
```

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by fin444** [comment]
Looks like an upstream issue. Was supposedly fixed in 13.0, but has a bug report in 14.0 as well.
https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42041
https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/43242

**Comment by amalgame21** [comment]
> Looks like an upstream issue. Was supposedly fixed in 13.0, but has a bug report in 14.0 as well. https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42041 https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/43242

Yes I noticed the bug report but it doesn't explain why it only affect `librewolf` but not `librewolf-unwrapped` and `firefox` on my NixOS machine.

The temporary fix for me is use `librewolf-unwrapped` and maunally create `$HOME/.local/share/applications/librewolf.desktop` file to make it appear in application launcher.

**Comment by amalgame21** [comment]
It is fixed by using a new profile in `librewolf` (with unknown reason).
`librewolf-unwrapped` works because it creates a new profile.

My work around is launching `librewolf` with `-P` flag and create a new profile, then quit librewolf without opening the browser.
Then in `~/.librewolf`, transfer all the files from the old profile to the new profile.
Then use `librewolf -P` again, select the new profile, then it mysteriously solve the problem.
